### ISSUE TITLE
Accessing Admin Settings

### ISSUE THREAD

**Issue by CopyJosh** [wpsight/wpcasa#16]
For the gist of the issues with wpcasa, see this:

https://wordpress.org/support/topic/insufficient-admin-permissions-on-activate?replies=4

Unfortunately I am having a problem viewing the settings pages for ALL the add-on applications.. ninja-forms, etc. At the moment, I cannot figure out how to resolve it.

**Comment by simonrimkus** [comment]
Hi Josh,

Thanks for your feedback!

In WPCasa we do not touch the administrator rights except that we add a few capabilities https://github.com/wpsight/wpcasa/blob/master/wpcasa/includes/class-wpsight-install.php#L44

I was not able to reproduce this issue nor have I heard of another WPCasa user experiencing something similar.

Do you have the chance to test on a different server with a fresh install?

Best regards,
Simon [WPCasa]

**Comment by CopyJosh** [comment]
So on a good note, the other issues I were having were resolved and I found the -personal- problem I was creating for myself.. oops.

Back to this "Listings" appearance on activation, I added the plugin to a live Wordpress install, and you're right... I didn't have any issues. I am not sure why this is a problem I'm replicating on my local dev servers.. somehow it's not resolving that plugin directory on it's own without the __DIR__ inserted, but indeed seems to be another personal problem. :(
